doc/tutorials/dnn/dnn_high_level_api/dnn_classification_module.markdown

```diff
@@ -0,0 +1,61 @@
+# High Level API: Classification Module {#tutorial_dnn_classification_module}
+
+### Introduction
+
+In this tutorial we will present how easy it is to run inference on a classification model using the new OpenCV Dnn API.
+
+### Usage
+
+Here is a simple snippet of the code in C++ with which you can run inference on a classification model.
+In this case the model is a simple AlexNet CNN  trained on ImageNet.
+
+~~~~~~~~~~~~~{.cpp}
+
+#include <opencv2/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/dnn.hpp>
+
+using namespace cv;
+using namespace dnn;
+
+int main()
+{
+
+    // Load our sample image
+    Mat image = imread(samples::findFile("ladybug.jpg"));
+
+    ClassificationModel model(samples::findFile("alexnet.onnx")); // create our model
+
+    // Define the transformations that we need to apply to our image
+    Size size{227, 227}; // resize
+
+
+    // Whether to swap Red and Blue channels since OpenCV loads images in BGR
+    bool swapRB = false;
+
+    // rescale the image (image * scale)
+    double scale = 1.0;
+    Scalar mean = Scalar(103.939, 116.779, 123.68); // mean to subtract to each channel
+
+    // Set the transformations we want to apply
+    model.setInputParams(scale, size, mean, swapRB);
+
+    // Network Forward pass
+    std::pair<int, float> prediction = model.classify(image);
+
+    std::string pred = "Prediction " + std::to_string(prediction.first);
+    std::string confidence = " Confidence " + std::to_string(prediction.second);
+
+    // It shows 301 which is the class for ladybug in ImageNet
+    imshow(pred + confidence, image);
+    waitKey(0);
+
+    return 0;
+}
+~~~~~~~~~~~~~
+
+### Results
+
+![Classification](images/ladybug_out.png)
```

Comment:
Please reuse existing OpenCV images. There are a lot of them: https://github.com/opencv/opencv/tree/master/samples/data